client/src/components/Home/Home.js

```diff
@@ -1,33 +1,11 @@
 import React, { Component } from 'react'
```

Comment:
Do we still need this component at all? I don't see it being used in the app anywhere...

Reply:
It was only added as my basic set up code for a component. However, I removed until I figure out what I am going to do with the Home component.

Reply:
Gotcha, no worries then, we can leave it for later use or remove it at a later time, no big deal.